test/e2e_node/memory_eviction_test.go

```diff
@@ -37,7 +37,9 @@ var _ = framework.KubeDescribe("MemoryEviction [Slow] [Serial] [Disruptive]", fu
 	f := framework.NewDefaultFramework("eviction-test")
 
 	Context("When there is memory pressure", func() {
-		It("It should evict pods in the correct order (besteffort first, then burstable, then guaranteed)", func() {
+		// Skip the test temporarily because of #30550.
+		// TODO(random-liu): Re-enable this test after #30550 is fixed.
+		PIt("It should evict pods in the correct order (besteffort first, then burstable, then guaranteed)", func() {
```

Comment:
@vishh @mtaufen I'm fine with #30476, and @coufon also verified that it worked. :)